packages/errors/src/__tests__/json-rpc-error-test.ts

```diff
@@ -20,6 +20,7 @@ import {
     SOLANA_ERROR__JSON_RPC__SERVER_ERROR_TRANSACTION_SIGNATURE_LEN_MISMATCH,
     SOLANA_ERROR__JSON_RPC__SERVER_ERROR_TRANSACTION_SIGNATURE_VERIFICATION_FAILURE,
     SOLANA_ERROR__JSON_RPC__SERVER_ERROR_UNSUPPORTED_TRANSACTION_VERSION,
+    SOLANA_ERROR__UNRECOGNIZED_JSON_RPC_ERROR,
```

Comment:
_Mayyyyybe_ `_MALFORMED_` to really land the accusation that these JSON-RPC servers are non-conforming.